src/pat/inject/inject.js

```diff
@@ -37,7 +37,7 @@ define([
 
     var _ = {
         name: "inject",
-        trigger: "a.pat-inject, form.pat-inject, .pat-subform.pat-inject",
+        trigger: ".ui-button.pat-inject, a.pat-inject, form.pat-inject, .pat-subform.pat-inject",
         init: function inject_init($el, opts) {
```

Comment:
i would like to use pat-inject on a div.ui-button element from raptor-editor toolbar. i would like to adjust raptor-editor's toolbar because there might be other issues with it.

what would be the best way to move on here?